src/server/system_services/schemas/agent_config_schema.js

```diff
@@ -31,9 +31,6 @@ module.exports = {
         use_storage: {
             type: 'boolean'
         },
-        use_s3: {
```

Comment:
what about upgrades?

Reply:
First I don't know if we have deployments with agents or that we need to take them into account.
Second, we can leave the definition. It is optional in any case.
Third, I can  create an upgrade script to delete all agent_configs (or just the field)

@nimrod-becker What od you think

Reply:
Assume systems with agents

Reply:
I don't think we should leave it just to avoid verification errors. just add an upgrade script